.github/workflows/tests.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       strategy:
         matrix:
           python-version: [ '3.9', '3.13' ]
-          os: [ "ubuntu-22.04", "ubuntu-24.04" ]
+          os: [ "ubuntu-22.04", "ubuntu-24.04", "windows-latest" ]
```

Comment:
Could you revert this line?
The tests will fail on Windows without intalling SWI-Prolog first.

Reply:
Done